app/views/reports/call_volumes.html.erb

```diff
@@ -18,6 +18,17 @@
 
       <div class="form-group">
         <%= f.submit 'Filter', class: 'btn btn-default t-filter-button' %>
+        <%=
```

Comment:
what are your thoughts on this given it is an internal tool as its not supported on ie 9 and below:

http://www.w3schools.com/tags/att_input_formaction.asp